src/wagtail_live/receivers.py

```diff
@@ -184,8 +184,11 @@ def process_text(self, live_post, message_text):
 
         message_parts = message_text.split("\n")
         for text in message_parts:
-            block_type = ""
+            # Avoid creating a block for empty content
+            if not text:
```

Comment:
Should we `strip` too? Because whitespace will evaluate as True, but still doesn't make sense.